tests/test_metadata.py

```diff
@@ -375,6 +378,50 @@ def test_disturl(
             )
 
 
+@pytest.mark.parametrize(
+    "container,container_name,container_type",
+    IMAGES_AND_NAMES_WITH_BASE_XFAIL,
+    indirect=["container"],
+)
+def test_readme_md_content(
+    container: ContainerData,
+    container_name: str,
+    container_type: ImageType,
+):
+    """General check of the ``io.artifacthub.package.readme-url`` label:
+
+    verify that if it exists, it is reachable and the content is valid
+    markdown.
+    """
+    labels = container.inspect.config.labels
+
+    readme_url = urlparse.urlparse(labels["io.artifacthub.package.readme-url"])
+
+    assert readme_url.scheme == "https"
+    assert readme_url.netloc in (
+        "api.opensuse.org",
```

Comment:
This one shouldn't be allowed, it's only reachable with credentials, which makes it useless for the use case of artifacthub

Reply:
api.opensuse.org has a /public route that does not require credentials and can be used for the use case of artifacthub